evals/elsuite/steganography/requirements.txt

```diff
@@ -0,0 +1,2 @@
+jiwer==3.0.2
```

Comment:
Similar to other PRs, I think this should be in https://github.com/openai/evals/blob/main/pyproject.toml